src/reformat_libyuv.c

```diff
@@ -60,6 +60,18 @@ unsigned int avifLibYUVVersion(void)
 #pragma clang diagnostic pop
 #endif
 
+// libyuv is a C++ library and defines custom types (struct, enum, etc) in a namespace when conditionally compiled with a C++
+// compiler. When accessed from a C library like libavif, via a function pointer, this leads to signature mismatches in the CFI
+// sanitizers since the C++ header file defines the types within the namespace and the C code has the types without the namespace.
```

Comment:
Nit/optional: We should change "the C++ header file defines the types within the namespace" to something like the following:

> libyuv itself, compiled as C++ code, has the types within the namespace

Reply:
Done.